nautobot/core/views/utils.py

```diff
@@ -284,26 +283,27 @@ def view_changes_not_saved(request, view, current_saved_view):
     If every configuration is the same, return False
     """
     if current_saved_view is None:
-        return False
+        return True
```

Comment:
I think this still should be False here because this is the case where the user is making modifications to the original list view.

Reply:
It is also not showing as intended because the if block for this boolean is wrapped in another one.
```
<h1>{% block title %}
    {{ title }}
        {% if current_saved_view %}
            — {{ current_saved_view.name }}
            {% if new_changes_not_applied %}
                <sup>*</sup>
            {% endif %}
        {% endif %}
    {% endblock %}
</h1>
```